mmv1/api/type.go

```diff
@@ -1354,3 +1357,9 @@ func (t Type) TGCSendEmptyValue() bool {
 
 	return false
 }
+
+// When the property has ignore_read: true and have custom_flatten method,
+// the custom_flatten method should not be applied when compiling the tgc-next provider.
```

Comment:
```suggestion
```
I think the name of this function is a good enough description of what it does and the comment is only adding confusion.